apps/sms/views/shared/test/unit/settings_test.js

```diff
@@ -159,7 +168,13 @@ suite('Settings >', function() {
         'operatorResource.sms.maxConcat',
         10
       );
+      triggerSettingsReqSuccess('messages.mms.sendReadReport.enabled', true);
+
+      Settings.sendReadReport.then(
+        (result) => assert.isTrue(result)
+      ).then(done, done);
 
+      sinon.assert.calledWith(Settings.initSendReadReport, true);
```

Comment:
answering to:
> yeah, But i think it's good to keep it as previous assertion check the promise return value and this one check that function is called with expected argument. Though it means the same.
Still, if you say, it will remove it then . @julienw

I think this assertion is testing something internal. In the end, what we want to know is whether sendReadReport returns the right value. We don't care how this is internally implemented.

Please tell me if you need clearer explanations, maybe I'm not clear :)